components-core/src/main/java/com/adyen/checkout/components/base/BaseConfigurationBuilder.kt

```diff
@@ -67,15 +67,14 @@ abstract class BaseConfigurationBuilder<
     }
 
     /**
-     * TODO docs
+     * Sets the amount of the transaction.
+     *
+     * Default is [Amount.EMPTY] for all configurations, except it is 0 USD for Google Pay Component.
```

Comment:
What do you think if we just write `Default is [Amount.EMPTY].` here, then override `setAmount` in google pay config builder and change the doc there, the method would only call super but the doc will be accurate.

Reply:
I thought we had the same case somewhere else but apparently I removed it by mistake, [re-added it here](https://github.com/Adyen/adyen-android/pull/983) if you wanna check what I mean.